Lib/test/test_typing.py

```diff
@@ -2535,6 +2535,100 @@ def meth(x): ...
         with self.assertRaises(TypeError):
             isinstance(C(), BadPG)
 
+    def test_protocols_isinstance_properties_and_descriptors(self):
+        class C:
+            @property
+            def attr(self):
+                return 42
+
+        class CustomDescriptor:
+            def __get__(self, obj, objtype=None):
+                return 42
+
+        class D:
+            attr = CustomDescriptor()
+
+        # Check that properties set on superclasses
+        # are still found by the isinstance() logic
+        class E(C): ...
+        class F(D): ...
+
+        for klass in C, D, E, F:
+            with self.subTest(klass=klass.__name__):
+                self.assertEqual(klass().attr, 42)
```

Comment:
Well, technically we are not testing this behaviour. So, these lines can be omitted, if we are doing tests "by the book".